modules/identity/pages/single-sign-on-with-oidc.adoc

```diff
@@ -177,7 +177,7 @@ NOTE: Activating this option means any user authorized by the IdP to access Boni
  ** `jwt-access-token` indicates if the access token is a JWT and should be validated. If false, it will be considered by Bonita OIDC module as an opaque token, its validation won't be performed (but the ID token will, however, be validated) and the retrieval of the logged in user principal name will be delegated to the userInfo endpoint (see `user-info-url`) when accessing a resource with a Bearer `Authorization` header. Even if the access token is a JWT, this property should be set to false when the OIDC clients are not supposed/able to validate it. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
  ** `scope-values` can be used to set the value of the `scope` parameter sent to the OIDC provider in the authentication request. The scope value `openid` is automatically added if it is no present, but you can add other scope values like `email` or `profile` depending on your OIDC provider configuration and the claims you need in the JWTs to match Bonita username. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
  ** Some OpenID Connect providers do not support having a pattern of valid redirecting URIs. It is the case with Google's for example. With those providers you can use the `state` parameter to pass the target URL (the URL you initially requested and want to be redirected to). In order to do that, set the property `save-target-url-in-state` to `true`. When you do that, the redirect_URI query parameter will always be the same. Its default value is the path `/bonita/keycloak` on Bonita server, but you can set a different value using the property `redirect-uri-path`. Make sure this URL is part of the valid redirect URIs in you OIDC provider configuration for the Bonita client. When `save-target-url-in-state` is set to `false` or not present, the `redirect_URI` query parameter is used to save the target URL. Those properties are specific to Bonita OIDC module and are not supported/documented by Keycloak.
- ** the `ssl-required` property value may need to be changed if Bonita Runtime and the IdP are not both accessed via HTTPS. Possible values for this property are: `all`, `external`, and `none`. For `all`, all requests must come in via HTTPS. For `external`, only non-private IP addresses must come over via HTTPS. For `none`, no requests are required to come over via HTTPS.
+ ** the `ssl-required` property value may need to be changed if Bonita Runtime and the IdP are not both accessed via HTTPS. Possible values for this property are: `all`, `external`, and `none`. For `all`, all requests must come in via HTTPS. For `external`, only non-private IP addresses must come over via HTTPS. For `none`, no requests are required to come over via HTTPS (not recommended for production).
```

Comment:
Azure AD requires SSL/HTTPS for Service Provider end point. Event if Bonita (Bundle or Container) is behind an HTTPS reverse proxy,  `ssl-required` should be set to `all`

Reply:
feedback added :+1: